tests/components/reolink/test_config_flow.py

```diff
@@ -495,3 +496,5 @@ async def test_dhcp_ip_update(
 
     await hass.async_block_till_done()
     assert config_entry.data[CONF_HOST] == expected
+
+    device_registry._data_to_save()
```

Comment:
This should not be needed in the test, I think it's a left over from debugging?

Reply:
True, I will remove it.
Had left it in to expose this problem in the future emidiatly when I change something else in the device_info and forget to add it to the mock.

Reply:
done

Reply:
I think in the future we should have a test for checking the devices in the device rehistry, which would've caught this

Reply:
I don't fully agree with this conclusion. The problem is that when tests end we shut down Home Assistant, which serializes the device registry. I would expect the tests to always fail, there must be something wrong with how we check the status when shutting down the tests.